convo_utility.py

```diff
@@ -0,0 +1,74 @@
+from conversation import *
+import chirps_utility
+from csv_utility import *
+from random_id_generator import *
+
+class ConvoUtility:
+
+	def new_public_convo(chirp_id, current_user):
+		"""
+		Creates a new public conversation based on a chirp id
+
+		Args- chirp id, current user
+		"""
+		# if a user is signed in
+		if current_user != None:
+			# create PublicConvo() with arguments
+			public_convo = PublicConvo(chirp_id, random_id_generator())
+			return public_convo
+		else:
+			print("\nPlease select a user first")
+
+	def new_private_convo(chirp_id, current_user):
+		"""
+		Creates a new public conversation based on a chirp id
+
+		Args- chirp id, current user
+		"""
+		# if a user is signed in
+		if current_user != None:
+			# create PrivateConvo() with arguments
+			private_convo = PrivateConvo(chirp_id, random_id_generator())
+			return private_convo
+		else:
+			print("\nPlease select a user first")
+
+	def show_public_reply_menu(chirp_list, convo_list, current_user):
+		"""
+		Allows users to reply to a selected chirp
+
+		Args-
+			list of all chirps,
+			conversation selected chirp is a part of,
+			currently selected user
+		"""
+		# print all chirps in provided convo
+		for id_num in convo_list:
+			chirp_info = chirp_dict[id_num]
```

Comment:
Attempting to reply to a public chirp results in `NameError: global name 'chirp_dict' is not defined`